.github/workflows/labels.yml

```diff
@@ -52,10 +52,7 @@ jobs:
         token: ${{ secrets.GITHUB_TOKEN }}
         issue-number: ${{ github.event.issue.number }}
         body: >
-          This issue looks more like a support question than an
-          issue. We strive to answer these reasonably fast, but
-          [purchasing the support subscription](https://weblate.org/support/)
-          is not only more responsible and faster for your business but also makes Weblate stronger.
+           This issue has been marked as a question by a Weblate team member. Why? Because it belongs more to the professional [Weblate Care](https://care.weblate.org/) or community [Discussions](https://github.com/WeblateOrg/weblate/discussions) than here. We strive to answer these reasonably fast here, too, but [purchasing the support subscription](https://weblate.org/support/) is more responsible and faster for your business. And it makes Weblate stronger as well. Thanks!
```

Comment:
```suggestion
          This issue has been marked as a question by a Weblate team member. 
          Why? Because it belongs more to the professional [Weblate Care](https://care.weblate.org/) 
          or community [Discussions](https://github.com/WeblateOrg/weblate/discussions) than here. 
          We strive to answer these reasonably fast here, too, but
          [purchasing the support subscription](https://weblate.org/support/) is more 
          responsible and faster for your business. And it makes Weblate stronger as well. Thanks!
```